lib/livebook/zta.ex

```diff
@@ -45,6 +45,16 @@ defmodule Livebook.ZTA do
   """
   @callback authenticate(name(), Plug.Conn.t(), keyword()) :: {Plug.Conn.t(), metadata() | nil}
 
+  @doc """
+  Logouts against the given name.
+  """
+  @callback logout(name(), Phoenix.LiveView.Socket.t()) :: :ok | :error | :unsupported
```

Comment:
We don't need `:unsupported`, since we just raise an exception instead :)

Also, for error perhaps it would make sense to have `{:error, String.t()}`, so we can show/log more information?